apps/guide/src/content/02-creating-your-bot/02-creating-commands.mdx

```diff
@@ -168,7 +172,7 @@ client.on('interactionCreate', async (interaction) => {
 	</DiscordMessage>
 </DiscordMessages>
 
-You could also display the date the server was created, or the server's verification level. You would do those in the same manner – use _`interaction.guild.createdAt`_ or _`interaction.guild.verificationLevel`_, respectively.
+You could also display the date the server was created, or the server's verification level. You would do those in the same manner—use _`interaction.guild.createdAt`_ or _`interaction.guild.verificationLevel`_, respectively.
```

Comment:
I'm not a fan of removing the spacing between the `-` here

Reply:
I replaced the en (`-`) dashes with em dashes (`—`) since the em dashes are used to denote extra information—but Github nor VSC seems to be rendering them correctly. The em dashes are mostly used without spaces therefore this change, since I not only changed the spacing but also the dash itself. It should look good on the website itself.

Reply:
Ehhh. 
![Google Chrome Dev - 2023-04-19 at 18 06 37@2x](https://user-images.githubusercontent.com/17960496/233118893-5f6f0c20-994d-4140-9728-83b6a72faae0.png)
I still don't like it but won't make it a blocker for the PR to be merged 🤷